app/.env.example

```diff
@@ -26,7 +26,7 @@ VUE_APP_NAMEX_API_VERSION="/api/v1"
 VUE_APP_NAMEX_API_KEY=
 VUE_APP_STATUS_API_URL="https://status-api-dev.apps.gold.devops.gov.bc.ca"
 VUE_APP_STATUS_API_VERSION="/api/v1"
-VUE_APP_REGISTRIES_SEARCH_API_URL="https://bcregistry-dev.apigee.net/registry-search"
+VUE_APP_REGISTRIES_SEARCH_API_URL="https://test.api.connect.gov.bc.ca/registry-search-dev"
```

Comment:
Fixed and tested locally:

<img width="709" height="187" alt="image" src="https://github.com/user-attachments/assets/e11b10a3-abdc-4fbd-bd2b-104222c73900" />

<img width="1080" height="476" alt="image" src="https://github.com/user-attachments/assets/afb4d974-de42-4ae4-9410-3f5499212bdc" />